src/providers/LayerProvider/LayerProvider.tsx

```diff
@@ -237,7 +235,6 @@ export const LayerProvider = ({
       params: { businessId },
     }),
     {
-      ...defaultSWRConfig,
```

Comment:
Removing the explicit setting of the default SWR config so it uses the single global config we define in the `SWRConfig`.